components/pipeline-service/production/base/pac-config.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- op: replace
+  path: /data/application-name
+  value: Red Hat Trusted App Pipeline
+- op: replace
+  path: /data/custom-console-name
+  value: Red Hat Trusted App Pipeline
+- op: replace
+  path: /data/custom-console-url
+  value: https://console.redhat.com/preview/hac/application-pipeline
```

Comment:
cc @christianvogt - aren't we going to remove "hac" from the URL path soon?

Reply:
`https://console.redhat.com/preview/application-pipeline` simply redirects to `https://console.redhat.com/preview/hac/application-pipeline` at this time.

@rohitkrai03 may know more on when it would be fully supported

Reply:
I guess that's good enough for us and I'll update the PR as it'll be better future-proof.